src/collections/events/service-mesh-and-meshery/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Journey Into the World of Service Meshes and Meshery"
+date: 2022-01-29
+thumbnail: ./service-mesh-and-meshery.png
+eurl: https://sched.co/siEs
+published: true
+upcoming: true
+speakers: ["Nithish Karthik", "Aditya Krishna Sharma"]
```

Comment:
Are we missing an "h", @adithyaakrishna?

Reply:
Yep, we are actually 😂 
Thanks for pointing it out @leecalcote ❤️